src/scripts/component/form.js

```diff
@@ -0,0 +1,65 @@
+// Curt
+
+import { populateHomepage } from "./homepage.js"
+
+//function to create the log in form
+function createLoginFormComponent () {
+    let formContainer = document.querySelector("#container")
+    formContainer.innerHTML = `
+    <h1>Welcome User. Please input your login information</h2>
+    <input id="userName" name="userEditor" type="text" placeholder="Your Name Here">
+    <input id="userPassword" name="userEditor" type="text" placeholder="Password">
+    <button id="loginBtn">Login</button>
+    `
+    loginEvent()
+}
+
+//function to create the registration form
+function createRegistrationForm() {
+    let formContainer = document.querySelector("#container")
+    formContainer.innerHTML = `
+    <h1>Welcome User. Please create user ID</h1>
```

Comment:
Could we get super fancy and welcome the user by identifier?

Reply:
no! too fancy!